### PR TITLE
Update localization README regarding pluralization

### LINKED ISSUE #9917: Pluralization doco
### Describe the bug
I'm trying to understand the doco for pluralization and I'm unclear how the T.Plural method is supposed to be used.

[Pluralization](https://docs.orchardcore.net/en/dev/docs/reference/modules/Localize/index.html#usage)

image

In this example, I believe that "1 book" is the singular form and "{0} books" is the plural form. Which form is used is decided based on the count parameter. Usually, this means that if `count == 1` the singular form is used, otherwise the plural form is used.

Now, let's say I want to run my OC in French.

[Wikipedia link for French rules](https://en.wikipedia.org/wiki/Grammatical_number#French)

image

If French treats zero as singular, then the rule changes such that that if `count == 0 || count == 1` the singular form is used, otherwise the plural form is used. However, the doco places a hardcoded "1" in the singular form. So doesn't that mean that if `count == 0`, the translated form of "1 book" is going to be displayed? And if the translator kept the hardcoded "1" the wrong number will be displayed.

I guess my question is, should the doco example actually be this?
```
T.Plural(count, "{0} book", "{0} books")
```

Maybe the doco is assuming the translator will replace the hardcoded "1" with "{0}"?

### PR DESCRIPTION
**Pull request by rjpowers10**
Fixes #9917

A while back I raised an issue about the guidance around using the Plural method. I think we shouldn't be hardcoding a "1" into the singular form because the singular form can be used for counts other than 1. French, for example, uses the singular form for both 0 and 1.

It didn't seem like a consensus was reached and then the discussion went quiet, so I'm submitting this draft PR.

### CODE REVIEW

**Comment by rjpowers10** [comment]
cc @hishamco

**Comment by hishamco** [comment]
Much better

**Comment by hishamco** [comment]
@rjpowers10 can we make this "Ready for review" to take it seriously, it took a long time :)

File: src/docs/reference/modules/Localize/README.md
Comment (Piedone) [inline src/docs/reference/modules/Localize/README.md:1]: Reading through https://github.com/OrchardCMS/OrchardCore/issues/9917 I learned a lot :). But we need others to have a quick understanding of what's happening too, why does the singular example include a parameter? Could you please add a single paragraph of explanation under `## Pluralization` about the differences between languages like English and French, and why you as a developer would use a parameter even for the singular form?

**Review by Piedone — CHANGES_REQUESTED** [review]

**Comment by Piedone** [comment]
Would you like to get back to this, @rjpowers10?

**Review by Piedone — APPROVED** [review]

**Comment by Piedone** [comment]
Thank you! Next time when you're done addressing all feedback of a review, click ["Re-request review"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#re-requesting-a-review) in the top-right corner for each reviewer when you're ready for another round of review, so they know that you're done.

**Pull request merged**